README.md

```diff
@@ -84,10 +84,20 @@ GuardDog comes with 2 types of heuristics:
 
 ### Running a local version of GuardDog
 
+#### Using pip
+
+* Ensure `>=python3.10` is installed
```

Comment:
@vdeturckheim purposely relaxed this requirement in a previous PR - any reason why we'd need 3.10 at least?

Reply:
Oh I see, that would be because of https://github.com/DataDog/guarddog/issues/93

Reply:
Yes, the scan codepath does not use any 3.10 syntax so my tests worked, but the verify path will break. We'll use docker images for lambdas

Reply:
So shall we revert and enforce 3.10 here as Zack suggests?